### ISSUE TITLE
"Java Not Found" error (Ubuntu says it's installed)

### ISSUE THREAD

**Issue by greped** [killpowa/Twickt-Launcher#1147]
I've checked the version of Java on Ubuntu, which says it's updated to JDK 16.0.2. I've since restarted the computer but the GDLauncher still returns the error "Java Not Found".

JDK 16.0.2 can be found with the path /usr/lib/jvm/

Is there something else I'm missing? A path or something I need to add?

**Comment by huantianad** [comment]
Make sure you’ve given gdlauncher the path to the java executable itself in the java settings

**Comment by greped** [comment]
> Make sure you’ve given gdlauncher the path to the java executable itself in the java settings

Is it not enough to set it to find the jvm folder?

**Comment by huantianad** [comment]
No, you need to give it the path to the executable. In the jvm folder, there should be a folder for the java version, then the executable will be at bin/java.

Gdlauncher doesn’t know which Java version to use if you just point it at the entire jvm folder, nor does it know where in the folder the executable is.

Hopefully in the future we have auto java detection for Linux
